README.md

```diff
@@ -14,26 +14,26 @@ See the [in-depth
 analysis](https://github.com/v8/web-tooling-benchmark/blob/master/docs/in-depth.md)
 for a detailed description of the tests included in this benchmark suite.
 
-The latest browser version of the benchmark is available at https://v8.github.io/web-tooling-benchmark.
+The latest browser version of the benchmark is available at <https://v8.github.io/web-tooling-benchmark/>.
```

Comment:
The canonical URL for the benchmark is <https://v8.github.io/web-tooling-benchmark/> and adding the slash to the link avoids an unnecessary HTTP redirect.